Tasks/UseDotNetV2/task.json

```diff
@@ -44,13 +44,31 @@
                 "sdk": "SDK (contains runtime)"
             }
         },
+        {
+            "name": "useGlobalJson",
+            "type": "boolean",
+            "label": "Use global json",
+            "defaultValue": false,
+            "required": false,
+            "helpMarkDown": "Select this option to install all SDKs from global.json files. These files are searched from system.DefaultWorkingDirectory. You can change the search root path by setting working directory input.",
+            "visibleRule": "packageType = sdk"
+        },
+        {
+            "name": "workingDirectory",
+            "type": "filePath",
+            "label": "Working Directory",
+            "helpMarkDown": "Specify path from where global.json files should be searched when using `Use global json`. If empty, `system.DefaultWorkingDirectory` will be considered as the root path.",
+            "required": "false",
+            "visibleRule": "useGlobalJson = true"
+        },
         {
             "name": "version",
             "type": "string",
             "label": "Version",
             "defaultValue": "",
-            "required": false,
-            "helpMarkDown": "Specify version of .NET Core SDK or runtime to install.<br/>Versions can be given in the following formats<li>2.x   => Install latest in major version.</li><li>2.2.x => Install latest in major and minor version</li><li>2.2.104 => Install exact version</li><br/>Find the value of `version` for installing SDK/Runtime, from the releases.json. The link to releases.json of that major.minor version can be found in [**releases-index file.**](https://github.com/dotnet/core/blob/master/release-notes/releases-index.json). Like link to releases.json for 2.2 version is https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/2.2/releases.json"
+            "required": true,
+            "helpMarkDown": "Specify version of .NET Core SDK or runtime to install.<br/>Versions can be given in the following formats<li>2.x   => Install latest in major version.</li><li>2.2.x => Install latest in major and minor version</li><li>2.2.104 => Install exact version</li><br/>Find the value of `version` for installing SDK/Runtime, from the releases.json. The link to releases.json of that major.minor version can be found in [**releases-index file.**](https://github.com/dotnet/core/blob/master/release-notes/releases-index.json). Like link to releases.json for 2.2 version is https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/2.2/releases.json",
+            "visibleRule": "useGlobalJson = false || packageType = runtime"
         },
         {
             "name": "includePreviewVersions",
```

Comment:
Do add a `visibleRule` for includePreviewVersions to hide this it if global.json input is selected. Saying so as includePreviewVersions has no significance when global.json is specified, and will only cause confusion if not hidden